python/pfnet/cpfnet.pyx

```diff
@@ -3326,11 +3705,27 @@ cdef class Network:
         def __get__(self):
             return [self.get_vargen(i) for i in range(self.num_var_generators)]
 
+    property var_gens:
+        """ List of network :class:`variable generators <pfnet.VarGenerator>` (list). """
+        def __get__(self):
+            if self.num_var_generators > 0:
```

Comment:
See point 5). We need to say that this is as alias, e.g. "same as ..." and we do not need the if statement.